pkgs/top-level/all-packages.nix

```diff
@@ -18387,6 +18387,10 @@ with pkgs;
   terraform = terraform_0_9;
 
   terragrunt = callPackage ../applications/networking/cluster/terragrunt {
+    terraform = terraform;
```

Comment:
You don't need the `terraform = terraform` here, you can just do `callPackage ../applications/networking/cluster/terragrunt {}`